src/index.css

```diff
@@ -209,6 +489,10 @@ blockquote {
 	margin: 1rem 0;
 }
 
+.dark .assistant-markdown hr {
+	border-top: 1px solid theme( 'colors.gray.700' );
+}
+
```

Comment:
Feels like there's a lot of CSS overrides for dark mode, I wonder if there's a way to avoid all of that by using CSS variables.

Reply:
Heavy +1. These CSS overrides will be incredibly difficult to maintain as we add new features. It also makes it harder to introduce new themes in the future (i.e. high contrast, custom user themes).